integration/utils/exec.go

```diff
@@ -410,12 +414,24 @@ func mountSetupFull(t *testing.T, stdout io.Writer, stderr io.Writer, user *Unix
 // If tests wish to control the shutdown of the sandboxfs process, they can do so, but then they
 // must set s.Cmd to nil to tell TearDown to not clean up the process a second time.  The same
 // applies to s.Stdin.
-func (s *MountState) TearDown(t *testing.T) {
+//
+// If tests wish to check if TearDown returned an error, they can do so by avoiding the recommended
+// use of "defer".  Note, though, that such tests will only receive the first error encountered by
+// this function, and that the function will run to completion even if there were failures.
+func (s *MountState) TearDown(t *testing.T) error {
+	var finalErr error
```

Comment:
Call this firstErr instead? 

That seems to be what it is, and the comment above talks about "first error" too

Reply:
Done. "final" here meant "error to be returned in the end".